src/themes/structural/modules/Button.ts

```diff
@@ -0,0 +1 @@
+export default {"button":"_button_312p3_1"};
```

Comment:
I ignored generated modules and added the `postcss` grunt task to both `dev` and `dist` to make sure it's run before use. I also added `/* tslint:disable:object-literal-key-quotes quotemark whitespace */` to the beginning of each generated module.